components/suggestion/search_channel_provider.jsx

```diff
@@ -21,6 +21,9 @@ function itemToName(item) {
     if (item.type === Constants.GM_CHANNEL) {
         return '@' + item.display_name.replace(/ /g, '');
     }
+    if (item.type === Constants.OPEN_CHANNEL || item.type === Constants.PRIVATE_CHANNEL) {
+        return item.display_name + ' ~(' + item.name + ')';
```

Comment:
the tilde character `~` should be close to the channel name, as it is mattermost indicator that it is a channel. If you look at the picture you added for mobile it is like that. Still, I'd like @esethna to confirm that this is the way we want to show to the user

Reply:
> this should only change the suggestion for channel autocompletion, so the user can identify a channel by either its id or its name, depending on what's familiar for him. With the proposed change it is adding extra info to the search and nothing can be found.
> 
> Let me know if you need any more details on the approach

are you saying this changes things besides the channel autocompletion and its breaking other things? it's not finding anything?